package.json

```diff
@@ -1,20 +1,21 @@
 {
-	"name": "krill",
-	"description": "simple boolean filter language",
-	"version": "0.3.0",
-	"license": "MIT",
-	"contributors": [
-	    "Robert Mustacchi <rm@joyent.com>",
-	    "Dave Pacheco <dap@joyent.com>"
-	],
-	"repository": {
-	    "type": "git",
-	    "url": "http://github.com/joyent/node-krill.git"
-	},
-	"main": "lib/krill.js",
-	"dependencies": {
-	    "extsprintf": "1.0.2",
-	    "jsprim": "0.5.1",
-	    "verror": "1.3.6"
-	}
+  "name": "krill",
```

Comment:
This is the result of running `npm install --save assert-plus` with npm 1.4.28. I don't know why the indentation changed, but I thought that might be because the previous file was generated by an older (major?) version of npm, and that we'd want to keep the style generated by a more "current" version of npm.

This may not be what happened, so that's a great question.